include/swift/AST/Module.h

```diff
@@ -1288,6 +1289,12 @@ class ModuleEntity {
   bool isSystemModule() const;
   bool isBuiltinModule() const;
   const ModuleDecl *getAsSwiftModule() const;
+  const clang::Module *getAsClangModule() const;
+
+  inline void *getOpaqueValue() const {
```

Comment:
`inline` is redundant, it's the fact that you have the definition present that makes it inlinable.